### ISSUE TITLE
Regarding kalerator.clueboard.co

### ISSUE THREAD

**Issue by Swoy** [skullydazed/kalerator#9]
Hello, every time I try to load a layout, I get internal server error. Am I missing something, or is the website deactivated?

**Comment by skullydazed** [comment]
It should still work. What's the URL of the layout you're trying to load?

**Comment by Swoy** [comment]
Yeah, I figured it out. When copying the address line "http" didn't get copied, and it required that to connect.

I apologize, I'm getting overly comfortable using shortcuts ^^

**Update**: Actually, the layout cannot have gaps it seems? And possibly also have to be set to public? Are there any place where I can read about limitations?

**Comment by Swoy** [comment]
http://www.keyboard-layout-editor.com/#/gists/d1a8d1a9c55f0b0fa0f16186ea78ddce

**Comment by skullydazed** [comment]
Ah, yeah, that would explain it. :)

At this point that can be simplified, but this software is in maintenance mode right now. I have plans for a refactor with improved functionality, but it's pretty low on my priority list for the foreseeable future.

**Comment by Swoy** [comment]
I wish I had time to help out :S Thanks anyway!
